platform-coop-toolkit/coop-map-directory-index
# feat: use Type.icon, create icon inclusion tag

## Description

This PR:

- Adopts `Type.icon` as seen in #111.
- Creates a Django [inclusion tag](https://docs.djangoproject.com/en/3.0/howto/custom-template-tags/#inclusion-tags) for generating SVG icon markup.

## Steps to test

1. Check out pull request.
2. Visit "My profiles" and a sample of organization and individual profile pages.

**Expected behavior:** All SVG icons load as expected.

## Additional information

Not applicable.

## Related issues

Blocked by #111.
